<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gps_Map_DockWidget</class>
 <widget class="QDockWidget" name="Gps_Map_DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gps Map</string>
  </property>
  <widget class="QWidget" name="Gps_Map_dockWidgetContents">
   <layout class="QGridLayout" name="Gps_Map_dockWidgetContents_gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="Map_groupBox">
      <property name="statusTip">
       <string>Map Group Box: Displays The Gps Map.</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="Map_groupBox_gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="Map_scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="Map_scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>257</width>
            <height>78</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="Map_scrollAreaWidgetContents_gridLayout">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="Options_groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="Options_groupBox_gridLayout">
       <item row="1" column="3">
        <widget class="QComboBox" name="Map_Type_comboBox">
         <property name="statusTip">
          <string>Map Type Combo Box: Changes Gps Map Display Type.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="Options_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Zoom_In_pushButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Zoom In Push Button: Zooms Into The Gps Map.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Zoom_Out_pushButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Zoom Out Push Button: Zooms Out Of The Gps Map.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
